contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -2,35 +2,428 @@
+          :label="$tr('conferenceSourcePlaceholder')"
+        />
+        <TextArea
+          v-else-if="form.source === sources.OTHER"
+          v-model="form.other_source"
+          :label="$tr('otherSourcePlaceholder')"
+        />
+
+        <!-- Policy -->
+        <ActionLink
+          class="mt-4"
+          :text="$tr('viewPrivacyPolicyLink')"
+          @click="showPolicies = true"
+        />
+        <VDialog v-model="showPolicies" width="500">
+          <VCard>
+            <VCardTitle class="headline grey lighten-4" primary-title>
+              {{ $tr('privacyPolicyTitle') }}
+            </VCardTitle>
+            <iframe :src="policyLink"></iframe>
+            <VDivider />
+            <VCardActions>
+              <VSpacer />
+              <VBtn color="primary" flat @click="showPolicies = false">
+                {{ $tr('closeButton') }}
+              </VBtn>
+            </VCardActions>
+          </VCard>
+        </VDialog>
+        <VCheckbox
+          v-model="form.accepted_policy"
+          :label="$tr('privacyPolicyCheck')"
+          color="primary"
+          required
+          :rules="policyRules"
+          class="mt-1 mb-3 policy-checkbox"
+        />
+
+        <p class="mb-4">
+          {{ $tr('contactMessage') }}
+        </p>
+        <VBtn color="primary" large type="submit">
+          {{ $tr('finishButton') }}
+        </VBtn>
+      </VForm>
+
+    </VLayout>
   </ImmersiveModalLayout>
 
 </template>
 
 
 <script>
 
+  import { mapActions } from 'vuex';
+  import { uses, sources } from '../constants';
+  import TextField from 'shared/views/form/TextField';
+  import EmailField from 'shared/views/form/EmailField';
+  import PasswordField from 'shared/views/form/PasswordField';
+  import TextArea from 'shared/views/form/TextArea';
+  import CountryField from 'shared/views/form/CountryField';
   import ImmersiveModalLayout from 'shared/layouts/ImmersiveModalLayout';
+  import Banner from 'shared/views/Banner';
 
   export default {
     name: 'Create',
     components: {
       ImmersiveModalLayout,
+      TextField,
+      EmailField,
+      PasswordField,
+      TextArea,
+      CountryField,
+      Banner,
+    },
+    data() {
+      return {
+        valid: true,
+        registrationFailed: false,
+        showPolicies: false,
+        emailErrors: [],
+        form: {
+          first_name: '',
+          last_name: '',
+          email: '',
+          password1: '',
+          password2: '',
+          uses: [],
+          storage: '',
+          other_use: '',
+          locations: [],
+          source: '',
+          organization: '',
+          conference: '',
+          other_source: '',
+          accepted_policy: false,
+        },
+      };
+    },
+    computed: {
+      passwordConfirmRules() {
+        return [v => this.form.password1 === v || this.$tr('passwordMatchMessage')];
+      },
+      policyRules() {
+        return [v => !!v || this.$tr('privacyPolicyRequiredMessage')];
```

Comment:
This function could be written more explicitly as:

```js
return [value => value ? true : this.$tr('privacyPolicyRequiredMessage')];
```

A side note: recommend avoiding `!!` as a way to cast something to a Boolean. Writing `Boolean(v)` is more explicit.